app/upgrades/v3/upgrades.go

```diff
@@ -0,0 +1,35 @@
+package v3
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	claimKeeper "github.com/Stride-Labs/stride/x/claim/keeper"
+	// claimTypes "github.com/Stride-Labs/stride/x/claim/types"
+)
+
+// Note: ensure these values are properly set before running upgrade
+const (
+	UpgradeName = "v3"
+	// airdropDistributor = "stride1htlzesdgm6pvunstrgk3p7aj6s66j08f9v3rgl"
```

Comment:
airdropDistributor is unique per each airdrop and its for sending claimable amounts to the user's vesting account.
I thought it would be a better approach for distributors setting up new airdrops whenever they needs but wasn't sure exactly so I left it commented out for your feedback.
Those three constant values are for only first airdrop after chain upgrade.
If you prefer to set a new airdrop automatically after chain upgrade, we can uncomment.
let me know your opinion.